src/ds/prob/hyperloglog.md

```diff
@@ -0,0 +1,62 @@
+In this tutorial you will learn how to use the HyperLogLog (HLL) probabilistic data structure in a bike shop use case.
+
+HyperLogLog is a probabilistic data structure that estimates the cardinality of a set. As a probabilistic data structure, HLL trades perfect accuracy for efficient space utilization.
```

Comment:
Maybe also add a layman's explanation for 'cardinality of a set' ('that is the number of unique elements ...')